pkg/apis/forklift/v1beta1/plan.go

```diff
@@ -264,6 +268,37 @@ func (p *Plan) ShouldUseV2vForTransfer() (bool, error) {
 	}
 }
 
+func (r *Plan) DestinationHasUdnNetwork(client k8sclient.Client) bool {
```

Comment:
we are abusing the api definitions file, this method is doing api calls using a k8s client, maybe we can put this method some ware else ?

Reply:
will it look better in adapter base ?